app/javascript/react/screens/App/Mappings/screens/MappingWizard/components/MappingWizardClustersStep/index.js

```diff
@@ -2,6 +2,7 @@ import { connect } from 'react-redux';
 
 import MappingWizardClustersStep from './MappingWizardClustersStep';
 import * as MappingWizardClustersStepActions from './MappingWizardClustersStepActions';
+import * as TargetResourcesActions from '../../../../../../../../redux/common/targetResources/targetResourcesActions';
```

Comment:
We probably don't want to import all the things as a general rule any more.  As we migrate more of the resource state management here, we'll only want to grab the appropriate actions.